projects/kit/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -209,4 +213,18 @@ export class TuiInputPhoneInternationalComponent
         this.countryIsoCode = code;
         this.countryIsoCodeChange.emit(code);
     }
+
+    private extractCountryCode(value: string): TuiCountryIsoCode | undefined {
+        return this.countries.find(countryIsoCode => {
+            const ruCodeTest =
```

Comment:
we have more cases.
For example, Japan starts with `81-##-####-####`

Let takes any `8 (812) xxx-xx-xx` (Saint-Petersburg).
Paste will not work too(

Reply:
oh, stop, sorry.

I have made checks on `taiga-ui.dev`, not feature stand

Reply:
you can add a test for this case

Reply:
yeah, i agree that adding some tests with phone numbers similar to these countries will be cool

```
    [TuiCountryIsoCode.CN]: {mask: '+86(###) ####-####', name: 'Китайская Н.Р.'},
    [TuiCountryIsoCode.HK]: {mask: '+852####-####', name: 'Гонконг'},
    [TuiCountryIsoCode.JP]: {mask: '+81-##-####-####', name: 'Япония '},
    [TuiCountryIsoCode.KH]: {mask: '+855##-###-###', name: 'Камбоджа'},
    [TuiCountryIsoCode.KP]: {mask: '+850####-#############', name: 'Корейская НДР'},
    [TuiCountryIsoCode.KR]: {mask: '+82##-###-####', name: 'Респ. Корея'},
    [TuiCountryIsoCode.LA]: {mask: '+856##-##-###-###', name: 'Лаос'},
    [TuiCountryIsoCode.MO]: {mask: '+853####-####', name: 'Макао'},
    [TuiCountryIsoCode.TW]: {mask: '+886#-####-####', name: 'Тайвань'},
    [TuiCountryIsoCode.VN]: {mask: '+84(###) ####-###', name: 'Вьетнам'},
```

Reply:
added for Taiwan and Rostov